### DISCUSSION TITLE
why the client.cpp in /examples can not be make to a exe in linux Ubuntu22.04

### DISCUSSION THREAD

**Discussion by MasterL-min** [uNetworking/uWebSockets#1645] [Q&A] [answered]
I use these instrutions to install uWebSockets:
```
git clone --recurse-submodules https://github.com/uNetworking/uWebSockets.git
sudo make 
sudo make install 
```
But i find that the client.cpp in /examples can not found to turn to a exe or something,is there some wrong i have made?
![022)K4Z1B5 IYHT53XY 75H](https://github.com/uNetworking/uWebSockets/assets/71004611/7e654bbe-a3d3-4b79-a2d7-305971fe9288)
![U(8)P_RLYB}}C59`(I{E22U](https://github.com/uNetworking/uWebSockets/assets/71004611/c157372c-9100-4c8a-9905-e185e89ad240)

**Comment by Nukoooo** [comment, marked as answer]
If you take a look at Client.cpp, you will find it says ``// This example is broken and doesn't do anything. It is a potential interface for a future potential client.``
